assets/scss/components/_report-error.scss

```diff
@@ -0,0 +1,43 @@
+/*
+Load Partial
+
+Styles to contain a page partial loaded into the current page by AJAX, and a loading spinner to be displayed while the content is being loaded.
+This is used on the 'Get help with this page' content on the templates of many front-ends.
+The component requires a [Javascript module](https://github.com/hmrc/assets-frontend/blob/master/assets/javascripts/modules/reportAProblem.js) in order to function.
```

Comment:
Similar to the second bullet point in https://github.com/hmrc/assets-frontend/issues/596